.travis.yml

```diff
@@ -3,12 +3,6 @@ node_js:
 - 6
 - 4
 
-matrix:
-  fast_finish: true
```

Comment:
Not only in that case. https://docs.travis-ci.com/user/customizing-the-build/#Fast-Finishing